app/helpers/dashboard_helper.rb

```diff
@@ -49,6 +49,17 @@ def count_reports(hosts)
     data
   end
 
+  def get_overview_json(report, options = {})
+    data = [
```

Comment:
I think the server should be agnostic to the format of the UI charting library, it should just return the data to be shown in a reasonable way.
that said, the chart infrastructure made by @gailsteiger is being used in the statistics page, and relies on arrays for values.
i would also prefer using objects, that are more descriptive, but we should have only one contract with the chart component. either array based or object based.